types/cloudflare/index.d.ts

```diff
@@ -95,7 +95,7 @@ declare namespace Cloudflare {
     }
 
     interface DnsRecordsBrowseResponse<RecordType extends RecordTypes> {
-        result: Array<DnsRecordByType<RecordType>> | null;
+        result: Array<DnsRecordByType<RecordType> & { id: string }> | null;
```

Comment:
You can include this new type in `DnsRecordByType` directly for cleanliness since it's only used in this one case.

Reply:
✅ 

I also updated the name to clarify its use case. Thanks!